src/main/java/bio/terra/cli/command/cromwell/generate.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+
+# Initialize a default Cromwell config. Don't overwrite in case the user has
+# customized their Cromwell config file on their PD.
+cromwell_config=/home/jupyter/cromwell/cromwell.conf
+echo "WORKSPACE_BUCKET: $1"
+echo "GOOGLE_PROJECT: $2"
+if [ ! -f "${cromwell_config}" ]; then
+  cat <<EOF | sudo -E -u jupyter tee "${cromwell_config}"
```

Comment:
Hi @yuhuyoyo I followed the AOU https://github.com/all-of-us/workbench/blob/71bd33f976dbe03b2a23fd1c588e826df00b1f26/api/src/main/webapp/static/initialize_notebook_runtime.sh seems like the sudo does not work in server. Do you have idea on it?

Reply:
can you try no sudo? the local process should already be running the script as the user.

also as discussed offline, set cromwell_config as an environment variable that can be specified by the user. Similar to what you did with GOOGLE_PROJECT, WORKSPACE_BUCKET and PET_SA_EMAIL.